sledger.go

```diff
@@ -37,18 +37,24 @@ const (
 
 func main() {
 	path := flag.String("ledger", "sledger.yaml", "path within git repository to sledger file")
-	database := flag.String("database", "postgresql://localhost", "URL of database to update")
+	var database string = *flag.String("database", "postgresql://localhost", "URL of database to update")
 	flag.Parse()
 
 	fmt.Println("==> Sledger")
 
 	fmt.Println("==> Parameters")
 	fmt.Println("  Ledger:", *path)
+	if isFlagPassed("database") {
+		fmt.Println("  Database:", database)
```

Comment:
In the case where the user passes the `--database` flag they are also likely to include the password.  Due to this, short of doing some nasty regex (which let's not do) I don't think we should print out whatever they passed in in order to protect the password.  Instead, let's just print out some static content saying that we received the string from the `--database` command line parameter and call it good.

Reply:
Agreed! Updated with latest commit